src/OpenTelemetry.Exporter.Prometheus.HTTP.Server/OpenTelemetry.Exporter.Prometheus.HTTP.Server.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@cijothomas @alanwest @reyang @utpilla How does everyone feel about `OpenTelemetry.Exporter.Prometheus.HTTP.Server` as the name? Something about it feels off to me. What I proposed on the issue was `OpenTelemetry.Exporter.Prometheus.HttpListener`. Don't _love_ that either, but I think it is more representative about what is going on.

Reply:
Given this is a `Draft` PR, I didn't make the comments but I do agree that `HTTP.Server` feels off.
I think `HttpListener` is _okay_, maybe `BasicHttpListener` or `SimpleHttpListener`?

Reply:
`OpenTelemetry.Exporter.Prometheus.HttpListener` is the better one of all.

The word simple "Simple" could be confused with "SimpleSpanProcessor" term from the spec... The doc (https://docs.microsoft.com/en-us/dotnet/api/system.net.httplistener?view=net-6.0) calls itself as "simple", so by using HttpListener, simplicity is implied..

The word "basic" could be incorrect when we add non-basic capabilities to that (in future)... 😄